install-universal-interfaces.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+#
+# install-universal-interfaces.sh
+#
+# Optionally download and attempt to install the universal interfaces from
+# a Macbinary disk image containing the Interfaces&Libraries directory
+#
+# Usage:
+#    install-universal-interfaces.sh <tempdir> <filename>
+#
+# Decompress the Macbinary image at <tempdir>/<filename> into
+# <tempdir>/InterfacesAndLibraries in the correct format for
+# interfaces-and-libraries.sh.
+#
+
+set -e
+
+MPWURL=https://www.ilande.co.uk/tmp/qemu/MPW-GM.img.bin
```

Comment:
Ah, another download location. Should this be here?

Reply:
Oops no, that was just for testing. Have repushed with that line removed.